backend/server/controllers/login.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
         }
       }).spread((user, created) => {
         const token = jwt.sign({ username: user.username, id: user.id }, process.env.SECRET)
+        logger.info('stuff', {user, token, created})
```

Comment:
Maybe change this to more descriptive log message?

Reply:
This shouldn't exist.